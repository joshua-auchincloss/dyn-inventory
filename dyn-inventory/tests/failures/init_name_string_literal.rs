// macro_name extraparams value must be an identifier not string literal
//
//
//
//
//
//
//
//
//
use dyn_inventory::dyn_inventory;

dyn_inventory!(
    LiteralRegistry<T: Plugin> {
        data: u64,
        t: T,
    };
    init_name = "MyInit",
);

fn main() {}
