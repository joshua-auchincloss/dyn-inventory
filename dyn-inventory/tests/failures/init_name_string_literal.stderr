error: expected identifier
  --> tests/failures/init_name_string_literal.rs:18:17
   |
18 |     init_name = "MyInit",
   |                 ^^^^^^^^
